Classes/ToxWrapper/OCTTox.h

```diff
@@ -389,6 +389,36 @@ extern const NSUInteger kOCTToxPublicKeyLength;
  */
 - (int)fileSendRequestWithFriendNumber:(int32_t)friendNumber fileName:(NSString *)fileName fileSize:(uint64_t)fileSize;
 
+/**
+ * TODO : Add info about method
+ */
+- (int)fileSendControlWithFriendNumber:(int32_t)friendNumber sendReceive:(uint8_t)sendReceive fileNumber:(uint8_t)fileNumber controlType:(uint8_t)controlType data:(const uint8_t *)data length:(uint16_t)length;
```

Comment:
It would be better to do more high-level wrapping:
- we can return `BOOL` here instead of -1 and 0
- `sendReceive` can be changed to `sendOrReceive:(OCTToxFileControlType)sendOrReceive`, with `OCTToxFileControlTypeSend` and `OCTToxFileControlTypeReceive` options
- `controlType` should use enum type that you've created
- `data` and it's `length` aren't used in Antidote, we can ignore and remove them for now.